producer/httprequest.go

```diff
@@ -78,9 +82,69 @@ func (prod *HTTPRequest) Configure(conf core.PluginConfigReader) error {
 	prod.encoding = conf.GetString("Encoding", "text/plain; charset=utf-8")
 	prod.rawPackets = conf.GetBool("RawData", true)
 
+	//prod.Log.Debug.Printf(
+	//"protocol=%s, host=%s, port=%s, address=%s, encoding=%s, rawPackets=%d",
+	//	prod.protocol, prod.host, prod.port, prod.address, prod.encoding, prod.rawPackets)
+
+	// Health check to ping the backend with an HTTP GET
+	healthcheck.AddEndpointPathArray(
```

Comment:
IMO making as little assumptions about plugins as possible ( == implicitly allowing multiple endpoints here) is more valuable than providing shorter syntax. 

From a new developer's perspective, diving through the initialisation code to find out how & where the endpoints are defined via the interface trick could also be more time-consuming than just looking up the single row where the plugin calls tgo.healthcheck.AddEndpoint(). 

And the healthcheck.AddEndpoint() approach would let us easily add health checks anywhere in the code, e.g. somewhere in the core classes if such a need comes up in the future.

But I agree, let's look at the details & relative merits IRL.